src/components/ContentNode/BlockVideo.vue

```diff
@@ -36,6 +36,21 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import 'docc-render/styles/_core.scss';
+
+.asset {
+  margin-left: auto;
+  margin-right: auto;
+
+  @include anything-before {
+    margin-top: $stacked-margin-large;
```

Comment:
Are we purposefully using "large" for the before spacing and "xlarge" for the after spacing here?

Reply:
No... 😵‍💫